web/src/css/main.scss

```diff
@@ -0,0 +1,22 @@
+@external _popup.scss;
```

Comment:
what's `@external`?
`@import` would be the way to include scss files in one another